### ISSUE TITLE
关于objectdetection

### ISSUE THREAD

**Issue by lzJune** [lwCVer/LWGANet#23]
您好，您论文中关于 对DOTA1.0/1.5的实验中，得到的mAP是多尺度实验得来的吗，先把trainval在mmrotate的split工具进行多尺度切片，然后再把官方的test数据集按同样方式多尺度切片再进行test.py吗，我想知道多尺度实验中需要对test数据集也进行多尺度切片吗？

**Comment by lwCVer** [comment]
您好，感谢您对我们工作的关注。论文中DOTA是单尺度的。












luwei_ahu
***@***.***

安徽大学




---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2026年1月22日(周四) 中午12:02
收件人: ***@***.***&gt;;
抄送: ***@***.***&gt;;
主题: [AeroVILab-AHU/LWGANet] 关于objectdetection (Issue #23)


lzJune created an issue (AeroVILab-AHU/LWGANet#23)
 
您好，您论文中关于 对DOTA1.0/1.5的实验中，得到的mAP是多尺度实验得来的吗，先把trainval在mmrotate的split工具进行多尺度切片，然后再把官方的test数据集按同样方式多尺度切片再进行test.py吗，我想知道多尺度实验中需要对test数据集也进行多尺度切片吗？

**Comment by lzJune** [comment]
> 您好，感谢您对我们工作的关注。论文中DOTA是单尺度的。
> 
> 
> 
> 
> 
> 
> 
> 
> 
> 
> 
> 
> luwei_ahu
> ***@***.***
> 
> 安徽大学
> […](#)

你好，请问训练集也是单尺度切片吗？

**Comment by lwCVer** [comment]
是的，训练验证测试均是单尺度切分



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2026年1月22日(周四) 中午1:25
收件人: ***@***.***&gt;;
抄送: "Wei Lu ***@***.******@***.***&gt;;
主题: Re: [AeroVILab-AHU/LWGANet] 关于objectdetection (Issue #23)


lzJune left a comment (AeroVILab-AHU/LWGANet#23)
  
您好，感谢您对我们工作的关注。论文中DOTA是单尺度的。
 
luwei_ahu
 @.***
 
安徽大学
 …
  
你好，请问训练集也是单尺度切片吗？

**Comment by UnchangedOriInten** [comment]
Hello, author. I am very interested in your work. I would like to ask about the experiment on target detection for DOTA 1.0/1.5 in the paper. During the training of the model, was a single training set used, or was it the combination of the training set and the validation set? Additionally, I would like to know whether the final MAP is the offline result on the validation set or the result on the online test set?

**Comment by lwCVer** [comment]
Dear Researcher,&nbsp;
Thank you for your interest in our work and for reaching out!&nbsp;
To answer your questions regarding the DOTA 1.0/1.5 object detection experiments:&nbsp;


1. Training Data: During the training phase, we utilized the combination of both the training set and the validation set (train + val) to train the final model.&nbsp;


2. mAP Evaluation: The final mAP scores reported in the paper are the online testing results obtained by submitting our model's predictions on the test set to the official DOTA evaluation server.&nbsp;


I hope this clarifies your questions. Please feel free to let me know if you have any further inquiries or need more details regarding the implementation.&nbsp;


Best regards,
Wei Lu




卢维
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "AeroVILab-AHU/LWGANet"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2026年7月8日(星期三) 下午5:58
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [AeroVILab-AHU/LWGANet] 关于objectdetection (Issue #23)



UnchangedOriInten left a comment (AeroVILab-AHU/LWGANet#23)
 
Hello, author. I am very interested in your work. I would like to ask about the experiment on target detection for DOTA 1.0/1.5 in the paper. During the training of the model, was a single training set used, or was it the combination of the training set and the validation set? Additionally, I would like to know whether the final MAP is the offline result on the validation set or the result on the online test set?
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications, keep track of coding agent tasks and review pull requests on the go with GitHub Mobile for iOS and Android. Download it today!
